src/Roslyn.Diagnostics.Analyzers/UnitTests/AvoidOptSuffixForNullableEnableCodeTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Roslyn.Diagnostics.CSharp.Analyzers.CSharpAvoidOptSuffixForNullableEnableCode,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Roslyn.Diagnostics.Analyzers.UnitTests
+{
+    public class AvoidOptSuffixForNullableEnableCodeTests
+    {
+        [Fact]
+        public async Task RS0037_CSharp8_NullableEnabledCode_Diagnostic()
```

Comment:
Test is failing with `error AD0001: Analyzer 'Roslyn.Diagnostics.CSharp.Analyzers.CSharpAvoidOptSuffixForNullableEnableCode' threw an exception of type 'System.MissingMethodException' with message 'Method not found: 'Microsoft.CodeAnalysis.CSharp.NullableContextOptions Microsoft.CodeAnalysis.CSharp.CSharpCompilationOptions.get_NullableContextOptions()'.'.`

The tests are using Roslyn 3.5 so I think that everything shall be ok.

Reply:
@sharwell Any idea what could be causing this `MissingMethodException`?